### ISSUE TITLE
SkypeForBusiness Powershell moudle requires .NET Framework 4.7?

### ISSUE THREAD

**Issue by lovelangy** [MicrosoftDocs/OfficeDocs-SkypeForBusiness#691]
https://docs.microsoft.com/en-us/skypeforbusiness/set-up-your-computer-for-windows-powershell/download-and-install-the-skype-for-business-online-connector

The document says it will install 2) .NET Framework 4.5.  That is not the case.
Based on my testing, the "SkypeOnlinePowerShell.exe" installation file contains: 

1.  vcredist_x64

2. NetFx47Redist(.NET Framework 4.7 not 4.5)

3. TenantPowershell

I am wondering:  is  .NET Framework 4.7 really needed by "SkypeOnlinePowerShell modules" or it is a just a package issue?  I am asking because Azure OS family 5 doesn't have .NET 4.7 installed.

If it is really required, i would suggest updating this document.

**Comment by AndreaBarr** [comment]
@lovelangy Thank you for this feedback

Thank you for pointing this out to us. I am going to get this to the Teams/Skype team so we can have this investigated and the document updated appropriately. 

I was able to get this into the backlog for the Teams team. The team will get this prioritized into a sprint plan. Thank you for reporting and making the docs better. Much appreciated.

I made a note to request the team to update this when the work is complete.
